components/compliance-service/inspec/types.go

```diff
@@ -42,6 +42,7 @@ type Control struct {
 	Refs           *json.RawMessage             `json:"refs"`
 	Tags           *json.RawMessage             `json:"tags"`
 	Results        []*reportingapi.Result       `json:"results,omitempty"`
+	StringTags     []*json.RawMessage           `json:"string_tags"`
```

Comment:
🤷‍♀ i kinda went with json rawmsg here cause it covers everything, but happy to change that if ppl want